subprojects/shipkit/src/main/groovy/org/shipkit/internal/notes/format/DetailedFormatter.java

```diff
@@ -60,20 +62,21 @@ public String formatReleaseNotes(Collection<ReleaseNotesData> data) {
         return sb.toString().trim();
     }
 
-    static String header(String version, Date date, boolean emphasizeVersion) {
-        return emphasizeVersion ? buildHeader(version, date, "# ", "")
-                : buildHeader(version, date, "**", "**");
+    static String header(String version, boolean emphasizeVersion) {
+        return emphasizeVersion ? buildHeader(version, "# ", "")
+            : buildHeader(version, "#### ", "");
     }
 
-    private static String buildHeader(String version, Date date, String prefix, String postfix) {
-        return prefix + version + " (" + DateUtil.formatDate(date) + ")" + postfix + " - ";
+    private static String buildHeader(String version, String prefix, String postfix) {
+        return prefix + version + postfix;
     }
 
-    static String releaseSummary(String version, ContributionSet contributions, Map<String, Contributor> contributors,
-                                 String vcsCommitsLink, String publicationRepository) {
-        return authorsSummary(contributions, contributors, vcsCommitsLink) +
-                " - published to " + getBintrayBadge(version, publicationRepository) + "\n" +
-                authorsSummaryAppendix(contributions, contributors);
+    static String releaseSummary(Date date, String version, ContributionSet contributions, Map<String, Contributor>
+        contributors, String vcsCommitsLink, String publicationRepository) {
+        return " - " + "(" + DateUtil.formatDate(date) + ") "
```

Comment:
" - " + "(" can be just " - (". 
Also the format is different than the one suggested by the issue submitted. It is:
`(2017-11-20) n commits`

and should be:
`2017-11-20 - n commits`

Let's put this date prefix into private method.